molecule/default/requirements.yml

```diff
@@ -1,3 +1,8 @@
----
```

Comment:
The three dashes should really be in the `requirements.yml` file.  This is a minor oversight in the skeleton-ansible-role PR.

Reply:
Also carelessness on my part; fixed in 922d027